cmd/get.go

```diff
@@ -6,16 +6,23 @@ package cmd
 import (
 	"fmt"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"path"
+	"path/filepath"
 	"strings"
 
 	"github.com/alexellis/arkade/pkg/env"
 	"github.com/alexellis/arkade/pkg/get"
+	"github.com/alexellis/arkade/pkg/helm"
 	"github.com/spf13/cobra"
 )
 
+func IsArchive(url string) bool {
+	return strings.HasSuffix(url, "tar.gz") || strings.HasSuffix(url, "zip")
```

Comment:
We can't support .zip at this time. I would remove that code.

Also `IsArchive` could be part of the `Tool` type, what do you think?

Reply:
Updated.